vignettes/histogram.Rmd

```diff
@@ -34,7 +34,7 @@ useTheme(vignetteTheme)
 
 The following vignette aims at documenting and illustrating workflows for producing histograms using the function `plotHistogram` from the `tlf` package.
 
-# 2. Illustration of basic PK ratio plots
+# 2. Illustration of basic historams
```

Comment:
typo: histogram

Reply:
Oh no ! My keyboard drives me nuts, there are some letters that are picky and are not necessarily typed when I press the key.
Just fixed it